test/system/libs/helpers.bash

```diff
@@ -61,6 +77,7 @@ function _pull_and_cache_distro_image() {
   fi
 
   for ((i = ${num_of_retries}; i > 0; i--)); do
+    echo $PODMAN pull ${image} >> test.log
```

Comment:
I'm unsure what this line is supposed to do. Could you explain, please?

Reply:
That was something I did to take a look at the output of podman pull and I forgot to reset to previous state.

:shame: